.github/drafts/15938/15938_Audit_Logs_Outline.md

```diff
@@ -0,0 +1,113 @@
+# Audit logs
+
+Audit logs are a vital way to keep records of user activity related to app changes and platform settings. With access to this information, you can ensure that you're able to easily follow up and address any unexpected or undesirable events.
+
+---
+
+{Screenshot of audit logs page}
+
+You can find audit logs by navigating to your Admin Settings page, and finding the **Audit Logs** tab under the **Others** heading. From here, you can see all the events that have been tracked under your organization's account, along with the user, date, and time that the event was logged. Click the arrow next to any event to expand it and see its full set of details in JSON form.
+
+## Logged events
+
+There are several types of events tracked in Appsmith's audit logs:
+
+- App-specific activity
+    - Changes to pages/widgets
+    - Datasource configuration
+    - Running queries
+    - Deleting apps
+
+- Admin settings changes
+    - Who changed certain settings
+
+| Event Name | Description |
```

Comment:
@spark-c Event details are for the user's reference, which should ideally come after the log format as mentioned in one of my earlier comments. 

So let's rearrange this as below(headings shared below are just to understand the overall structure)

- Intro along with what kinds of events (can be hyperlinked to the event details reference) are logged -App-specific activity, Admin settings changes,Sensitive information access, User sign-up and login activity,..
- Where to access Audit Logs on the Appsmith platform - with a screenshot
- Audit Log Format - example of the JSON with the properties and a brief explanation about those
- Tracked Events - Table of events along with the description
- Filtering Audit Logs

@vuiets does this look good? I noticed you mentioned that the filtering audit logs section should also move up, but this is an additional feature we are providing, not the highlight of the audit logs feature, correct? please do suggest if you feel this needs to change :)

Reply:
Moved the table to a subheading "Tracked Events" below the Example JSON snippet and pushed the commit, take a look and let me know if either of you would like changes :)